### PR TITLE
GNY-322 update to enable client credentials

### PR DESCRIPTION
**Pull request by HilaryLynch-expanse**
## Description

https://qadium.atlassian.net/browse/GNY-322

## Motivation and Context

Add support for Client credentials auth flow and updated the Readme

## How Has This Been Tested?

Tested locally

## Screenshots (if appropriate)

Added warning for bearer token

## Types of changes

- New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes if appropriate.
- [x] All new and existing tests passed. (see PR 106)

## Follow up
Tests will be done in a follow up PR. Will not merge these changes until that PR is reviewed.

### CODE REVIEW

File: xpanse/api/annotations/v3/tags.py [hunk 1]
```diff
@@ -82,7 +81,7 @@ def create(
             >>> # Create a new tag
             >>> tag = client.annotations.tags.v3.create(name="DMZ")
         """
-        payload = {"name": name, "description": description, "disabled": disabled}
+        payload = {"name": name, "description": description}
```
Comment (HilaryLynch-expanse) [inline xpanse/api/annotations/v3/tags.py:84]: This doesn't support passing in "disabled=True" on create any longer.

File: xpanse/api/issues/v1/issues.py [hunk 2]
```diff
@@ -394,6 +394,7 @@ def bulk_update(
                 {
                     "issueId": id_,
                     "updateRequest": {"value": value, "updateType": updateType},
+                    "operationType": "Update"
```
Comment (HilaryLynch-expanse) [inline xpanse/api/issues/v1/issues.py:397]: New field needed for bulk updates (Update is the only one right now)

File: README.md [hunk 3]
```diff
@@ -54,29 +54,33 @@ You can view more example code in the `examples <https://github.com/PaloAltoNetw
...
     # or
     export XPANSE_JWT_TOKEN=<JWT>
 
+[Deprecated]
+A valid Bearer token or JWT is required for use. Bearer tokens are suggested as JWTs have a limited lifespan.
 
-NOT RECOMMENDED
-***************
-You can also provided these keys directly at client initialization by doing
+RECOMMENDED
```
Comment (andrew-paloalto) [inline README.md:74]: We may want to clean this up a bit, it's suggesting `XPANSE_JWT_TOKEN` twice and mentioned bearer tokens can be used in one location but not in the other.

Comment (HilaryLynch-expanse) [reply README.md:74]: Good catch 👍

**Review by cachukwu — DISMISSED** [review]

File: xpanse/__init__.py [hunk 4]
```diff
@@ -1,6 +1,6 @@
-__version__ = "1.0.12"
+__version__ = "1.1.0"
 __author__ = "Palo Alto Cortex Xpanse"
 __license__ = "ISC"
 __maintainer__ = "Palo Alto Cortex Xpanse"
 __email__ = "help@expanseinc.com"
-__status__ = "Development"
+__status__ = "Production"
```
Comment (HilaryLynch-expanse) [inline xpanse/__init__.py:6]: @cachukwu forgot to bump this version, and made the status "Production". I don't think we use this status anywhere in the docs, but updated to keep it consistent with the status of the project.

Comment (HilaryLynch-expanse) [reply xpanse/__init__.py:6]: I'm going to merge these changes( only update since your last review)

**Pull request merged**